publisher/config/environments/test.rb

```diff
@@ -0,0 +1,36 @@
+Rails.application.configure do
```

Comment:
do we need a test environment for this app? are there tests?

Reply:
it's not really a standalone app, so I couldn't write normal tests. though I could write a test to exercise the release code. then the rake task would just be calling a PORO that does the shell stuff.

Reply:
I'd be in favor of having at least a basic test here that evaluates your error messages and statuses etc. (Edit: moved to this comment)